### DISCUSSION TITLE
why can't my laragon use nginx and I was given a solution to change as shown below, what is the problem with other applications?

### DISCUSSION THREAD

**Discussion by SATRIAARROZAK** [leokhoa/laragon#938] [Q&A]
![Screenshot 2024-12-18 144458](https://github.com/user-attachments/assets/b67778b7-072f-491d-a073-48b8542fd7b4)

**Comment by husnilkhatimi** [comment]
Its port conflict, when more than one app want to use the same port number 

Follow the given step so that Skype release the port, and can be use by Nginx
